test/irb/test_history.rb

```diff
@@ -488,6 +488,28 @@ def foo
       HISTORY
     end
 
+    def test_direct_debug_session_loads_history
+      @envs['RUBY_DEBUG_IRB_CONSOLE'] = "1"
+      write_history <<~HISTORY
+        old_history_1
+        old_history_2
+        old_history_3
+      HISTORY
+
+      write_ruby <<~'RUBY'
+        require 'debug'
+        debugger
+        binding.irb # needed to satisfy run_ruby_file
+      RUBY
+
+      output = run_ruby_file do
+        type "history"
+        type "exit!"
+      end
+
+      assert_include(output, "old_history_3")
```

Comment:
The current test already captures what we want to cover, but I think we can make expectations even clearer to help future readers understand the context:

```suggestion
      output = run_ruby_file do
        type "history"
        type "puts 'foo'"
        type "history"
        type "exit!"
      end

      assert_include(output, "irb:rdbg(main):002") # assert that we're in an irb:rdbg session
      assert_include(output, "5: history")
      assert_include(output, "4: puts 'foo'")
      assert_include(output, "3: history")
      assert_include(output, "2: old_history_3")
      assert_include(output, "1: old_history_2")
      assert_include(output, "0: old_history_1")
```

Reply:
Nice additions!